src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,44 @@
+package duke;
+import java.io.FileNotFoundException;
+
+/**
+ * Main class for Duke application.
+ */
+public class Duke {
+
+    private TaskList tasklist;
+    private Storage storage;
+    private TaskList tasks;
```

Comment:
Maybe you will want to comment out or delete this line because I do not see the variable `tasks` used anywhere, and you already have a variable `tasklist` for it.